### PR TITLE
Improve docs for sharing logic in parent and child features.

### PR DESCRIPTION
**Pull request by mbrandonw**

### CODE REVIEW

File: Sources/ComposableArchitecture/Documentation.docc/Articles/Performance.md [hunk 1]
```diff
@@ -382,6 +382,40 @@ store.send(.textFieldChanged("Hello") {
 }
 ```
 
+##### Sharing logic in child features
+
+There is another common scenario for sharing logic in features where the parent feature wants to
+invoke logic in a child feature. One can technically do this by sending actions from the parent 
+to the child, but we do not recommend it:
```
[the remark this answers is not in the document]
Comment (mbrandonw) [reply Sources/ComposableArchitecture/Documentation.docc/Articles/Performance.md:389]: Well, the _why_ is everything above these additions, which you can find here:

https://pointfreeco.github.io/swift-composable-architecture/main/documentation/composablearchitecture/performance#Sharing-logic-with-actions

Do you think it needs to be repeated?

[the remark this answers is not in the document]
Comment (jshier) [reply Sources/ComposableArchitecture/Documentation.docc/Articles/Performance.md:389]: Ah, then a link might be in order.Given people can be linked directly to sections in these docs, linking to the other sections with explanations seems like a good idea.

[the remark this answers is not in the document]
Comment (mbrandonw) [reply Sources/ComposableArchitecture/Documentation.docc/Articles/Performance.md:389]: Yeah that's a good idea, will do.

File: Sources/ComposableArchitecture/Documentation.docc/Articles/Performance.md [hunk 2]
```diff
@@ -382,6 +382,28 @@ store.send(.textFieldChanged("Hello") {
...
+case .buttonTapped:
+  // Send action to child to perform logic:
+  return .send(.child(.refresh))
+```
+
+Instead, we recommend invoking the child reducer directly:
+
+```swift
+case .buttonTapped:
+  return Child().reduce(into: state.child, action: .refresh)
```
Comment (stephencelis) [inline Sources/ComposableArchitecture/Documentation.docc/Articles/Performance.md:403]:
```suggestion
  return Child().reduce(into: &state.child, action: .refresh)
```

**Review by stephencelis — APPROVED** [review]

**Pull request merged**